docs/servo_power_monitor/src/index.ts

```diff
@@ -26,6 +26,9 @@ let halt = false;
 
 let currentData = undefined;
 function updateGraph(data: Array<Array<Date|number>>) {
+  if (data !== undefined && data.length) {
```

Comment:
If you just need a sanity check, probably just `!data` is fine. If you want a proper check, probably want to call out explicitly that you want `.length` to be a positive number. The typing probably has you covered but no harm in doing that vs just doing a truthy test against length. (Especially since truthy testing in JS is different from C/C++, which is the lingua franca of the other people who'd be touching this code.)